config/locales/models/en.yml

```diff
@@ -155,6 +155,13 @@ en:
               not_work: Only a link to a work can be listed as an inspiration.
               protected: You can't use the related works function to cite works by the protected user %{login}.
           format: "%{message}"
+        skin:
+          archive_in_title: You can't use the word 'Archive' in your skin title, sorry! (We have to reserve it for official skins.)
+          attributes:
+            title:
+              taken: must be unique
+          invalid_media: We don't currently support the media type %{media}, sorry! If we should, please let Support know.
```

Comment:
Second suggestion: update this to link "let Support know" to the feedback/support form